src/client/components/ContactForm/ContactForm.styles.css

```diff
@@ -0,0 +1,47 @@
+form {
+  display: grid;
+  padding: 1em;
+  background: #f9f9f9;
+  border: 1px solid #c1c1c1;
+  margin: 2rem auto 0 auto;
+  max-width: 400px;
+  padding: 1em;
+}
+
+label {
+  padding: 0.5em 0.5em 0.5em 0;
+}
+
+@media (min-width: 400px) {
+  form {
+    grid-template-columns: 1fr 8fr;
+    grid-gap: 6px;
+    font-style: normal;
+    font-family: 'Inter', sans-serif;
```

Comment:
The font-family should be assigned for the whole project, and therefore not specified here

Reply:
Thanks for the review, code adjusted as suggested.